cmd/sync/sync.go

```diff
@@ -12,40 +17,55 @@ import (
 	"github.com/jim-minter/azure-helm/pkg/plugin"
 )
 
-var dryRun = flag.Bool("dry-run", false, "Print resources to be synced instead of mutating cluster state")
+var (
+	dryRun = flag.Bool("dry-run", false, "Print resources to be synced instead of mutating cluster state.")
+	once   = flag.Bool("run-once", true, "If true, run only once then quit.")
```

Comment:
I am defaulting to `true` to "keep backwards-compatibility" with the current flow.

Reply:
Better to set the default right - the current flow is only half implemented and there is no need for backwards compatibility.

Reply:
In that case I guess I need to use the flag inside `./create.sh` so that I won't break it current usage? Trying not to boil the ocean in one go here

Reply:
In that case I guess I need to use the flag inside `./create.sh` so that I won't break its current usage? Trying not to boil the ocean in one go here.